account_company_currency/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    is_company_curr = fields.Boolean("Is_Company_Currency",
+        compute="_compute_is_company_curr",
+        store=True,
+    )
+    
+    @api.depends("company_currency_id")
```

Comment:
```suggestion
```
We can remove the trigger in case we do not store the value.

If we keep the trigger, we should include `currency_id` as well.